RecoVertex/V0Producer/src/V0Fitter.cc

```diff
@@ -168,8 +170,8 @@ void V0Fitter::fitAll(const edm::Event& iEvent, const edm::EventSetup& iSetup,
       if (sqrt(cxPt.x()*cxPt.x() + cxPt.y()*cxPt.y()) > 120. || std::abs(cxPt.z()) > 300.) continue;
 
       // the tracks should at least point in the same quadrant
-      TrajectoryStateClosestToPoint const & posTSCP = posTransTkPtr->trajectoryStateClosestToPoint(cxPt);
-      TrajectoryStateClosestToPoint const & negTSCP = negTransTkPtr->trajectoryStateClosestToPoint(cxPt);
+      TrajectoryStateClosestToPoint const posTSCP = posTransTkPtr->trajectoryStateClosestToPoint(cxPt);
```

Comment:
IIUC this change is not necessary

Reply:
IIRC `trajectoryStateClosestToPoint` returns also a copy of `TrajectoryStateClosestToPoint`. See: https://github.com/cms-sw/cmssw/blob/master/TrackingTools/TransientTrack/interface/BasicTransientTrack.h#L51

Thus I decided to remove reference to avoid potential issue. It's unrelated to the original report.

Reply:
it appears that in the other PR the point is that ```auto const& temp = a.returnByValue()```
will keep the temp in the scope and an explicit copy is not needed

Reply:
Ah yes, binding temporary to a const reference extends the it's lifetime.

This part is fine. Not needed, then @Dr15Jones PR looks better.

Reply:
thanks for confirming.
please close this PR.